Modules/Core/Common/include/itkImageHelper.h

```diff
@@ -62,73 +61,55 @@ class ImageHelper
 
   /** ComputeIndex with recursive templates */
   inline static void
-  ComputeIndex(const IndexType &     bufferedRegionIndex,
-               OffsetValueType       offset,
-               const OffsetValueType offsetTable[],
-               IndexType &           index)
+  ComputeIndex(const IndexType &                      bufferedRegionIndex,
+               OffsetValueType                        offset,
+               [[maybe_unused]] const OffsetValueType offsetTable[],
```

Comment:
Note that this pull request uses the C++17 attribute `[[maybe_unused]]`, to avoid warnings appearing at the `else` part of the "constexpr if".